apps/api/src/app/import/import.service.ts

```diff
@@ -470,14 +419,43 @@ export class ImportService {
         });
       });
 
+      const dataGatheringItems = uniqueActivities.flatMap(
```

Comment:
I have reverted this part. It was not working properly and we don't gather exchange rates automatically e.g. when you create an activity manually. I would prefer to have it in one place.